net/foolsm/files/foolsm.conf

```diff
@@ -0,0 +1,43 @@
+#
+# (C) 2009 Mika Ilmaranta <ilmis at nullnet.fi>
+#
+# License: GPLv2
```

Comment:
I suggest you complain to upstream about this, but it's not something I'm going to be upset about here.

Reply:
Sorry, I'm not seeing the problem.  Complain upstream about what?

Reply:
well, IMO, having the _config_ file be GPL makes it pretty hard for anyone to use it, but that's something for upstream.  Like I said, I don't care about this package per se.

Reply:
This is [from OpenWRT history](https://dev.openwrt.org/browser/packages/net/lsm/files/lsm.conf) as well.